app/src/main/java/com/alphawallet/app/entity/TokensMapping.java

```diff
@@ -0,0 +1,59 @@
+package com.alphawallet.app.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class TokensMapping {
+
+    public class Contract {
+
+        @SerializedName("address")
```

Comment:
Can we remove @SerializedName when the names between JSON and Java object are identical?
https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/annotations/SerializedName.html

Reply:
@seabornlee can you add a refactor PR with these changes? I will push this PR as-is

Reply:
Sure.